app/components/audio/song-control.tsx

```
@@ -114,22 +114,24 @@
     <div className={twMerge(
       'song-playing z-[1000] bg-[--md-sys-color-inverse-on-surface] flex-col transition-transform duration-300',
       shouldHide && 'hidden'
-    )}>
-      <div className="p-4 gap-4 flex flex-wrap items-center justify-between">
+    )} onClick={() => {
+      if (window.innerWidth < 768 && currentSong?.id) router.push(`/song/${currentSong.id}`)}
+    }>
```

Biome (1.9.4)

[error] Enforce to have the onClick mouse event with the onKeyUp, the onKeyDown, or the onKeyPress keyboard event.

Actions triggered using mouse events should have corresponding keyboard events to account for keyboard-only navigation.

(lint/a11y/useKeyWithClickEvents)